config/thresholds/mtr-2026.yml

```diff
@@ -1,5 +1,7 @@
 ---
 # MTR (Means Test Review) config file - start date TBD
+# Mark test-only so that it can't
```

Comment:
I feel like this comment is not complete and if required should be completed and if not it should be removed